google/cloud/storage/blob.py

```diff
@@ -3031,19 +3032,24 @@ def compose(
             Can also be passed as a tuple (connect_timeout, read_timeout).
             See :meth:`requests.Session.request` documentation for details.
 
-        :type if_generation_match: list of long
+        :type if_generation_match: long
         :param if_generation_match:
             (Optional) Make the operation conditional on whether the blob's
             current generation matches the given value.  Setting to 0 makes the
             operation succeed only if there are no live versions of the blob.
-            The list must match ``sources`` item-to-item.
 
         :type if_metageneration_match: list of long
         :param if_metageneration_match:
             (Optional) Make the operation conditional on whether the blob's
             current metageneration matches the given value. The list must match
             ``sources`` item-to-item.
 
+        :type if_source_generation_match: list of long
+        :param if_source_generation_match:
+            (Optional) Make the operation conditional on whether the source blob's
```

Comment:
Given that `sources` is expected to be plural, this should likely expect that here, e.g.:

```(python)
            (Optional) Make the operation conditional on whether the current generation
            of each source blob matches the corresponding generation.
            The list must match ``sources`` item-to-item.
```